src/Plutus/Certification/API/Routes.hs

```diff
@@ -265,11 +262,34 @@ instance ToJSON DAppBody where
     , "githubToken"  .= dappGitHubToken
     ]
 
+newtype Twitter = Twitter { unTwitter :: Text }
+
+instance FromJSON Twitter where
+    parseJSON = withText "Twitter" $ \v ->
+      -- validate twitter account regex format ^[A-Za-z0-9_]{1,15}$
+      if not (isTwitterValid v)
+        then fail "Invalid twitter account format"
+        else pure $ Twitter v
+
+isTwitterValid :: Text -> Bool
+isTwitterValid v = not (
+  Text.length v > 15 || Text.length v < 1 || Text.any (\c -> not (isAlphaNum c || c == '_')) v
+  )
+
+instance ToJSON Twitter where
+  toJSON (Twitter v) = String v
+
+instance ToSchema Twitter where
+  declareNamedSchema _ = do
+    return $ NamedSchema (Just "Twitter") $ mempty
+      & type_ ?~ SwaggerString
+      & SL.pattern ?~ "^[A-Za-z0-9_]{1,15}$"
+
 data ProfileBody = ProfileBody
    { dapp :: !(Maybe DAppBody)
-   , website :: !(Maybe Text)
+   , website :: !(Maybe BaseUrl)
    , vendor :: !(Maybe Text)
-   , twitter :: !(Maybe Text)
+   , twitter :: !(Maybe Twitter)
    , linkedin :: !(Maybe Text)
```

Comment:
Can we also sanitize linkedin to ensure that if it is filled, it is indeed to a linkedin page?

If I give it a try (just for fun, not a recommendation in any way):
````haskell
linkedInPattern :: Pattern
-- | Straight outta ChatGPT
linkedInPattern = "^(?:https?:\/\/)?(?:www\.)?(?:linkedin\.com\/(?:in|pub|company)\/|lnkd\.in\/)(?:(?!.*?\/)[a-zA-Z0-9_-]+)(?:\/.*)?$" 


instance ToSchema LinkedIn where
		declareNamedSchema _ = do 
			return $ NamedSchema (Just "LinkedIn") $ mempty
			& type_ $~ SwaggerString
			& SL.pattern ?~ linkedInPattern
````

Reply:
done. please take a look at the latest commit 91b67b9d741c47511737ec7a720a53f11e384f9e